.github/workflows/build.yml

```diff
@@ -245,15 +202,15 @@ jobs:
           fi
 
       ########## ACRs ##########
-      - name: Login to Azure - PROD Subscription
```

Comment:
❓ Is the environment uppercased for a reason? In many places I've worked before that's been a common convention.